NEWS.md

```diff
@@ -1,5 +1,12 @@
 # MpiIsoApp development version
 
+## Version 23.02.3
+
+### Updates
+- Option to decide if column with duplicate rows is added
+- Option to ignore spaces during duplicate detection
```

Comment:
Could there be a bug here?

Without "Ignore Spaces" (1):
![image](https://user-images.githubusercontent.com/16759098/218761124-294bac9e-39a5-43bc-a11f-dcfc81a97e9b.png)

With "Ignore Spaces" (2):
![image](https://user-images.githubusercontent.com/16759098/218761559-560a0ffb-8c61-46d0-8c60-01eea34c8200.png)